tachyon/c/math/matrix/baby_bear_row_major_matrix.h

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file baby_bear_row_major_matrix.h
+ * @brief Row major matrix operations for the baby bear.
```

Comment:
Please change all instances to either "for the baby bear field" or "for baby bear"